packages/compass-sidebar/src/typings.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module '@mongodb-js/mongodb-redux-common/app-registry';
```

Comment:
Copy/pasted this from elsewhere, but it is probably about time that we type that module?

Reply:
Amazing how new modules still pops up after years .. does someone know why is that not part of this repo?

Also it seems to only have 1 file, maybe can be moved in another package that already exists? https://github.com/mongodb-js/redux-common/blob/master/app-registry.js